bayesian/utils.py

```diff
@@ -624,15 +627,17 @@ def _check_usage_outlier(self, pkg_name, manifest_response):
 
     def _companion_reason(self, manifest_response):
         """
-        It will populate the simple reason for each companion package
+        It will populate the simple reason for each companion package.
```

Comment:
please follow https://www.python.org/dev/peps/pep-0257/ so we don't have to manually do everything.

Reply:
@pkajaba I did not write the doc-string, just updated it as per our `doc-string checker`.

Reply:
yes, but if you change ones which you encounter (which is fairly easy) we end up with better coverage and less work in future.